report/sale_report_template.xml

```diff
@@ -5,8 +5,15 @@
             <t t-set="doc" t-value="doc.with_context(lang=doc.partner_id.lang)"/>
             <t t-set="forced_vat" t-value="doc.fiscal_position_id.foreign_vat"/> <!-- So that it appears in the footer of the report instead of the company VAT if it's set -->
             <t t-set="address">
-                <div t-field="doc.partner_id" t-options="{&quot;widget&quot;: &quot;contact&quot;, &quot;fields&quot;: [&quot;address&quot;, &quot;name&quot;], &quot;no_marker&quot;: True}"/>
-                <p t-if="doc.partner_id.vat"><t t-esc="doc.company_id.account_fiscal_country_id.vat_label or 'Tax ID'"/>: <span t-field="doc.partner_id.vat"/></p>
+                <div t-field="doc.partner_id" t-options="{&quot;widget&quot;: &quot;contact&quot;, &quot;fields&quot;: [&quot;address&quot;, &quot;name&quot;, &quot;nif&quot;], &quot;no_marker&quot;: True}"/>
+                <p t-if="doc.partner_id.country_id.code != 'CD'">
+                    <p t-if="doc.partner_id.vat">
+                        <t t-esc="doc.company_id.account_fiscal_country_id.vat_label or 'Tax ID'"/>: <span t-field="doc.partner_id.vat"/></p>
+                </p>
+                <!-- Add nif / rccm / id nat in [image]Quotation/Order -->
+                <p t-if="doc.partner_id.country_id.code == 'CD' and doc.partner_id.nif">NIF: <span t-field="doc.partner_id.nif"/></p>
+                <p t-if="doc.partner_id.country_id.code == 'CD' and doc.partner_id.rccm">RCCM: <span t-field="doc.partner_id.rccm"/></p>
+                <p t-if="doc.partner_id.country_id.code == 'CD' and doc.partner_id.id_nat">Id Nat: <span t-field="doc.partner_id.id_nat"/></p>
```

Comment:
Do we need the "doc.partner_id.country_id.code == 'cd', as the nif is defined only id the partner is CD...
I think you should only keep the t-if "<div t-if="o.partner_id.id_nat">". What do you think ?

Reply:
It's make sense👍